src/httprpc.cpp

```diff
@@ -147,8 +149,135 @@ static bool RPCAuthorized(const std::string& strAuth, std::string& strAuthUserna
     return multiUserAuthorized(strUserPass);
 }
 
+static bool checkCORS(HTTPRequest* req)
+{
+    // https://www.w3.org/TR/cors/#resource-requests
+
+    // 1. If the Origin header is not present terminate this set of steps.
+    // The request is outside the scope of this specification.
+    std::pair<bool, std::string> origin = req->GetHeader("origin");
+    if (!origin.first) {
+        return false;
+    }
+
+    // 2. If the value of the Origin header is not a case-sensitive match for
+    // any of the values in list of origins do not set any additional headers
+    // and terminate this set of steps.
+    // Note: Always matching is acceptable since the list of origins can be
+    // unbounded.
```

Comment:
@eugenekatsov Can you add an explicit note here with the 

// syscoin 
piece that notes explicitly why this is disabled?  I know why and you know why, but the next maintainer may not.